README.md

```diff
@@ -1 +1,31 @@
-# school-library-OOP
\ No newline at end of file
+# school-library-OOP
+This is an Educational project for the Microverse Module 3 Block 4(Ruby) School library exercise, requirement for the Ruby Module.
+
+Built With
+Ruby
+Pre-requisites
+You need to have Ruby installed on your local machine
+
+Ruby bundler properly set up for installing dependencies.
+
+
+Git clone https://github.com/asnganje/school-library-OOP.git 
+
+
+## Authors
+👤 **Abdulrahman Nganje**
+
+- GitHub: [@Nganje](https://github.com/asnganje)
+- LinkedIn: [@Nganje](https://www.linkedin.com/in/abdulrahman-nganje-a6436935/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse Inc
```

Comment:
- [ ] Please make sure that your readme has a link to a license to your repo and also has a link to an issue in your repo for contribution to make your readme more professional. :smile: